libs/community/langchain_community/vectorstores/sqlserver.py

```diff
@@ -199,3 +199,39 @@ def _insert_embeddings(
         except DBAPIError as e:
             logging.error(e.__cause__)
         return ids
+
+    def delete(self, ids: Optional[List[str]] = None, **kwargs: Any) -> Optional[bool]:
+        """Delete embeddings in the vectorstore by the ids.
+
+        Args:
+            ids: List of IDs to delete.
+            kwargs: vectorstore specific parameters.
+
+        Returns:
+            Optional[bool]
+        """
+
+        if ids is None or len(ids) == 0:
+            logging.info("Empty list of ids provided")
+            raise ValueError("No ids provided to delete.")
+
+        result = self.delete_texts_by_ids(ids)
+        if result == 0:
+            logging.info("Provided ids not found in the DB.")
```

Comment:
Shouldn't this be logging.warn or error?

Reply:
I did not add it as log.error/log.warn, as it is still a successful call to the DB and no failure has occurred, only data was not found which is a valid case. I added the log for future debug purpose. 

Do you suggest me to change it to log.error.

Reply:
I understand now, logging.info makes sense here..